projects/core/src/util/regex-pattern.ts

```diff
@@ -2,3 +2,5 @@
 export const EMAIL_PATTERN = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/; // tslint:disable-line
 
 export const PASSWORD_PATTERN = /^(?=.*?[A-Z])(?=.*?[0-9])(?=.*?[!@#$%^*()_\-+{};:.,]).{6,}$/;
+
+export const GUID_PATTERN = '[0-9a-f]{40}|[0-9a-f-]{36}';
```

Comment:
Is this configurable in the backend?

Reply:
Asking because if someone change it in the backend this wont work

Reply:
It is not property, but set in `ycommercewebservices-spring.xml`.